src/rpc/blockchain.cpp

```diff
@@ -1160,36 +1155,28 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"difficulty\": xxxxxx,         (numeric) the current difficulty\n"
             "  \"mediantime\": xxxxxx,         (numeric) median time for the current best block\n"
             "  \"verificationprogress\": xxxx, (numeric) estimate of verification progress [0..1]\n"
-            "  \"initialblockdownload\": xxxx, (bool) (debug information) estimate of whether this node is in Initial Block Download mode.\n"
```

Comment:
Curious why removal of this key and others (e.g. `size_on_disk`) is bundled with this changeset. Seems like RPC interface changes should be handled separately.

Reply:
Oops. Bad rebase. Should be fixed now.